### PR TITLE
Changed the What's New icon

### LINKED ISSUE #2552: Change the icon of subscription update
There are 2 actions with the same icon in NewPipe:
1) going to outer RSS feed has real RSS logo as an icon (https://en.wikipedia.org/wiki/RSS)

2) subscription update has the same icon but opens new video list in NewPipe

I think it would be better to change second icon from real RSS logo to something more abstract (like this one), it should me more comfortable.

### PR DESCRIPTION
**Pull request by Marius1501**
#### What is it?
- [ ] Bugfix (user facing)
- [x] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- Changed the What's New icon 
- Now it has a Youtube-styling

#### Before/After Screenshots/Screen Record

NewPipe2 – ic_whats_new xml  NewPipe2 app main  13 01 2023 16_37_10

#### Fixes the following issue(s)

- Fixes #2552

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: app/src/main/res/drawable/ic_whats_new.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (Stypox) [inline app/src/main/res/drawable/ic_whats_new.xml:1]: This icon is called `subscriptions` [here](https://fonts.google.com/icons?selected=Material+Icons&icon.query=subscriptions), so please keep the original name for consistency

**Review by Stypox — CHANGES_REQUESTED** [review]

**Comment by tvnmguy** [comment]
There is some changes needs;

[1]

[2]

[3]

[4]

**Comment by Marius1501** [comment]
@tvnmguy all the changes you mentioned are already in this PR except for the naming of the What's New and the Subscriptions section. Should we really rename these sections?

**Comment by Stypox** [comment]
>  Should we really rename these sections?

No, "what's new" is a perfectly valid name. "Subscriptions" and "Channels" might be confused instead.

**Review by Stypox — APPROVED** [review]
Works, thank you!

**Pull request merged**